src/main/kotlin/no/nav/familie/ks/sak/kjerne/brev/BrevService.kt

```diff
@@ -81,27 +101,50 @@ class BrevService(
         manueltBrevRequest: ManueltBrevDto,
         erForhåndsvisning: Boolean = false
     ): ByteArray {
-        Result.runCatching {
+        try {
             val brev = manueltBrevRequest.tilBrev()
             return brevKlient.genererBrev(
                 målform = manueltBrevRequest.mottakerMålform.tilSanityFormat(),
                 brev = brev
             )
-        }.fold(
-            onSuccess = { it },
-            onFailure = {
-                if (it is Feil) {
-                    throw it
-                } else {
-                    throw Feil(
-                        message = "Klarte ikke generere brev for ${manueltBrevRequest.brevmal}. ${it.message}",
-                        frontendFeilmelding = "${if (erForhåndsvisning) "Det har skjedd en feil" else "Det har skjedd en feil, og brevet er ikke sendt"}. Prøv igjen, og ta kontakt med brukerstøtte hvis problemet vedvarer.",
-                        httpStatus = HttpStatus.INTERNAL_SERVER_ERROR,
-                        throwable = it
-                    )
-                }
+        } catch (it: Exception) {
```

Comment:
runCatching fanger opp throwable, som betyr at out of memory feil og lignende blir fanget av denne. Dette ønsker vi å kaste videre opp til spring

Reply:
Enig men da bør vi også skrive om `handleException` i eksterntjenestekaller også siden den wrapper exceptions fra klientene